src/components/index.ts

```diff
@@ -4,7 +4,6 @@
  */
 
 // external shared components
-export { BaseAddress } from '@bcrs-shared-components/base-address'
```

Comment:
BaseAddress is directly imported wherever it's used, so no need to re-export it here. (And actually, I think it's better to directly import it.)